LayoutTests/platform/gtk/accessibility/custom-elements/describedby-shadow-expected.txt

```diff
@@ -0,0 +1,32 @@
+This tests that aria fallback roles work correctly.
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+
+PASS internals.ariaDescribedByElements.length is 2
+PASS internals.ariaDescribedByElements[0] is document.querySelectorAll(".note")[0]
+PASS internals.ariaDescribedByElements[1] is document.querySelectorAll(".note")[1]
+PASS internals.ariaDetailsElements.length is 2
+PASS internals.ariaDetailsElements[0] is document.querySelectorAll(".details")[0]
+PASS internals.ariaDetailsElements[1] is document.querySelectorAll(".details")[1]
+PASS internals.ariaErrorMessageElements.length is 2
+PASS internals.ariaErrorMessageElements[0] is document.querySelectorAll(".error")[0]
+PASS internals.ariaErrorMessageElements[1] is document.querySelectorAll(".error")[1]
+PASS accessibilityController.accessibleElementById("custom-1").role is "AXRole: AXCheckBox"
+PASS accessibilityController.accessibleElementById("custom-1").helpText is "AXHelp: some description other description"
+FAIL accessibilityController.accessibleElementById("custom-1").detailsElements().length should be 2. Threw exception TypeError: undefined is not an object (evaluating 'accessibilityController.accessibleElementById("custom-1").detailsElements().length')
+FAIL accessibilityController.accessibleElementById("custom-1").detailsElements()[0].domIdentifier should be details1. Threw exception TypeError: undefined is not an object (evaluating 'accessibilityController.accessibleElementById("custom-1").detailsElements()[0]')
+FAIL accessibilityController.accessibleElementById("custom-1").detailsElements()[1].domIdentifier should be details2. Threw exception TypeError: undefined is not an object (evaluating 'accessibilityController.accessibleElementById("custom-1").detailsElements()[1]')
+FAIL accessibilityController.accessibleElementById("custom-1").errorMessageElements().length should be 2. Threw exception TypeError: undefined is not an object (evaluating 'accessibilityController.accessibleElementById("custom-1").errorMessageElements().length')
+FAIL accessibilityController.accessibleElementById("custom-1").errorMessageElements()[0].domIdentifier should be error1. Threw exception TypeError: undefined is not an object (evaluating 'accessibilityController.accessibleElementById("custom-1").errorMessageElements()[0]')
+FAIL accessibilityController.accessibleElementById("custom-1").errorMessageElements()[1].domIdentifier should be error2. Threw exception TypeError: undefined is not an object (evaluating 'accessibilityController.accessibleElementById("custom-1").errorMessageElements()[1]')
```

Comment:
These shouldn't be needed after https://commits.webkit.org/254726@main, you can hopefully remove this WebKitGTK `-expected.txt` file.